src/components/mine/landing-page/footer.tsx

```
@@ -14,77 +17,127 @@ import {
+         
+        </div>
+
+        <div className=" p-4 border-b border-foreground/10 ">
+         
+          <div className="grid grid-cols-[1fr_1fr_2fr] w-fit  gap-4 font-medium text-foreground/70 ">
+
+            <ul>
+
+              {
+                firstLI.map((el, id) => {
+                  return <Link href={el.href}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)